Update and rename mattermost-communication.md

natalie-hub opened the pull request:
Rename to to mattermost-communication-values.md
Also updated formatting and added additional text, bullets

<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

natalie-hub committed:
Update and rename mattermost-communication.md 

Rename to to mattermost-communication-values.md
Also updated formatting and added additional text, bullets

jasonblais commented on operations/workplace/mattermost-communication-values.md:
```diff
@@ -2,10 +2,14 @@
-## Mattermost Communication Guidelines
+## Mattermost Communication Values
+
+As an open source, remote-only and communities-centered company, effective, efficient and courteous communication is expected of every team member. A few guidelines we live by:  
```
```suggestion
As an open source, remote-first and communities-centered company, effective, efficient and courteous communication is expected of every team member. A few guidelines we live by:  
```

jasonblais reviewed (changes requested):
propose 'remote-only' -> 'remote-first'

natalie-hub committed:
Update operations/workplace/mattermost-communication-values.md

Co-authored-by: Jason Blais <13119842+jasonblais@users.noreply.github.com>

natalie-hub committed:
Rename mattermost-communication-values.md

Rename from "Communication Values" to "Remote Values" 

Also, changed header for "Communication Values" to "Remote Values." I realized not all of the points were about communication. Also, may be worth developing more tangible day-to-day communication guidelines for clarity.

natalie-hub reviewed (commented):
Thanks for the updates @jasonblais
